tests/emb6/Makefile

```diff
@@ -41,4 +40,5 @@ USEMODULE += $(DRIVER)
 CFLAGS += -Wno-unused-parameter -Wno-unused-function -Wno-type-limits
 CFLAGS += -Wno-sign-compare -Wno-missing-field-initializers
 
+include ../Makefile.tests_common
```

Comment:
Same here as `tests_at86rf2xx`, we are using `BOARD` for configuring so would still prefer keeping the include before. I know a default value is defined, but I would prefer to keep it coherent.